projects/plugins/jetpack/modules/calypsoify/class-jetpack-calypsoify.php

```diff
@@ -87,7 +88,8 @@ public function enqueue_for_gutenberg() {
 			'calypsoifyGutenberg',
 			array(
 				'closeUrl'                => $this->get_close_gutenberg_url(),
-				'manageReusableBlocksUrl' => $this->get_calypso_origin() . '/types/wp_block/' . ( new Status() )->get_site_suffix(),
+				'manageReusableBlocksUrl' => $this->get_calypso_origin() . '/types/wp_block/' . $site_suffix,
+				'createNewPostUrl'        => $this->get_calypso_origin() . '/post' . $site_suffix,
```

Comment:
Shouldn't there be an additional `/` here?

```suggestion
				'createNewPostUrl'        => $this->get_calypso_origin() . '/post/' . $site_suffix,
```

Reply:
🤦‍♂️ thanks for catching that, now updated